File: messaging-service/src/main/java/com/yazikochesalna/messagingservice/config/properties/ChatServiceProperties.java
package com.yazikochesalna.messagingservice.config.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Component
@Data
@ConfigurationProperties(prefix = "chat.service")
public class ChatServiceProperties {
    private String url;


}

File: message-storage-service/src/main/kotlin/com/yazikochesalna/messagestorageservice/config/properties/ChatServiceProperties.kt
package com.yazikochesalna.messagestorageservice.config.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.boot.context.properties.bind.ConstructorBinding


@ConfigurationProperties(prefix = "chat.service")
data class ChatServiceProperties(
    val url: String
)
